components/sysutils/borgbackup/history

```diff
@@ -0,0 +1 @@
+backup/borg-35@1.1.17-2020.0.1.1
```

Comment:
This should be "backup/borg-35@1.1.17,5.11-2020.0.1.1"
5.11 is the build version, typically left out when the FMRI is being printed (and separated by a "," from the component version). According to the docs the build version is the version of the OS the component is being built for.

Reply:
Sorry for that. I took FMRI from `pkg info borg-35` which now I see doesn't contain the whole version.

Reply:
Never mind, I made the same mistake in the past and it took some time to understand it.